doc/next/_static/doctools.js

```diff
@@ -91,30 +91,6 @@ jQuery.fn.highlightText = function(text, className) {
   });
 };
 
-/*
- * backward compatibility for jQuery.browser
- * This will be supported until firefox bug is fixed.
- */
-if (!jQuery.browser) {
```

Comment:
hmmm what's your version of sphinx

Reply:
```
$ sphinx-build --version
Sphinx (sphinx-build) 1.5.5
```

Reply:
I will merge anyhow, don;'t see that copyright 2014 exposed anywhere, and I seem to have the correct sphinx version.